swan-lake/integration-tutorials/sending-emails-from-a-service.md

```diff
@@ -95,171 +95,127 @@ Follow the instructions given in this section to develop the service.
-    > 
-    > Alternatively, the clients can be initialized directly with `new` expressions, but a separate function is used to aid with testing.
-    > 
-    > ```ballerina
-    > final http:Client hospitalServicesEP = check new (hospitalServicesBackend);
-    > final http:Client paymentEP = check new (paymentBackend);
-    > ```
+        ![Define the service](/learn/images/integration-tutorials/sending-emails-from-a-service/define_a_service.gif)
+     
+    - Define an HTTP resource that allows the `POST` operation on the resource path `/categories/{category}/reserve`, where `category` (corresponding to the specialization) is a path parameter.
 
-5. Define records corresponding to the request payload and response payloads.
+    - Use `ReservationRequest` as a parameter indicating that the resource expects a JSON object corresponding to `ReservationRequest` as the payload. 
 
-    ```ballerina
-    type Patient record {|
-        string name;
-        string dob;
-        string ssn;
-        string address;
-        string phone;
-        string email;
-    |};
-
-    type Doctor record {|
-        string name;
-        string hospital;
-        string category;
-        string availability;
-        decimal fee;
-    |};
-
-    type Appointment record {|
-        int appointmentNumber;
-        Doctor doctor;
-        Patient patient;
-        boolean confirmed;
-        string hospital;
-        string appointmentDate;
-    |};
-
-    type PaymentSettlement record {|
-        int appointmentNumber;
-        Doctor doctor;
-        Patient patient;
-        decimal fee;
-        boolean confirmed;
-        string card_number;
-    |};
-
-    type Payment record {|
-        int appointmentNo;
-        string doctorName;
-        string patient;
-        decimal actualFee;
-        int discount;
-        decimal discounted;
-        string paymentID;
-        string status;
-    |};
-
-    type ReservationRequest record {|
-        record {|
-            *Patient;
-            string cardNo;
-        |} patient;
-        string doctor;
-        string hospital_id;
-        string hospital;
-        string appointment_date;
-    |};
-
-    type ReservationResponse record {|
-        int appointmentNo;
-        string doctorName;
-        string patient;
-        decimal actualFee;
-        int discount;
-        decimal discounted;
-        string paymentID;
-        string status;
-    |};
-    ```
+    - Use `http:Created|http:NotFound|http:InternalServerError` as the return type to indicate that the response will be an `http:Created` response when the email is sent successfully to the user or the response will be an `http:NotFound` or `http:InternalServerError` response on error.
 
-   - The `ReservationRequest` record uses [record type inclusion](https://ballerina.io/learn/by-example/type-inclusion-for-records/) in the type of the `patient` field to include all the fields from the `Patient` record along with the `cardNo` field.
```

Comment:
Combine these 3 similar to Please change the flow similar to https://github.com/ballerina-platform/ballerina-dev-website/blob/integration-tutorials/swan-lake/integration-tutorials/sending-a-message-to-a-service.md?plain=1#L69 since they correspond to the GIF after.